web_development_101/javascript_basics/project_calculator.md

```diff
@@ -26,7 +26,7 @@ Here are some use cases (abilities your project needs to have):
    1. You should already have the code that can populate the display, so once `operate()` has been called, update the display with the 'solution' to the operation.
    2. This is the hardest part of the project. You need to figure out how to store all the values and call the operate function with them. Don't feel bad if it takes you a while to figure out the logic.
 6. Gotchas: watch out for and fix these bugs if they show up in your code:
-   1. Users should be able to string together several operations and get the right answer, with each pair of numbers being evaluated at a time. For example, `12 + 7 - 5 * 3 =` should yield `42`. An example of the behavior we're looking for would be [this online calculator](https://www.theonlinecalculator.com). **Note** that this behavior means your calculator is **not required** to follow order of operations, aka PEMDAS (though there's nothing stopping you from implementing PEMDAS if you feel like it!).
+   1. Users should be able to string together several operations and get the right answer, with each pair of numbers being evaluated at a time. For example, `12 + 7 - 5 * 3 =` should yield `42`. An example of the behavior we're looking for would be [this online calculator](https://www.theonlinecalculator.com). **Note** that this behavior means your calculator **should not** follow order of operations, aka PEMDAS (your calculator should work like the calculator in the example).
```

Comment:
I don't quite think this is accurate though. The calculator shouldn't let you be in a position to enter anything that doesn't follow order of operations.

We want the student to have this sort of workflow:

enter a number
enter an operator
enter another number
enter another operator -> calculator should do `operate(num1, num2, operator)` -> set `num1` to the calculated number, and set operator to the newly entered operator.



---


Also, the other part of the PR requirements was to change the calculator example link. We need one that behaves in the way described above. Like this one: https://i3uckwheat.github.io/JS-Calculator/
But prettier.